features/manage-position/Create.tsx

```diff
@@ -194,68 +191,76 @@ const Create = () => {
             them if you want to short the underlying.
           </Typography>
         </Box>
-        <Box py={2}>
+        <Box pb={2}>
```

Comment:
What does `pb` vs `py` do?

Reply:
`pb` is padding bottom. `py` is padding & bottom and top. see https://material-ui.com/system/spacing/#notation